docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -34,6 +34,54 @@ location. When launched manually, Gitea can be killed using `Ctrl+C`.
 ./gitea web
 ```
 
+## Recommended server configuration
+
+### Prepare environment
+
+Check that git is installed on the server, if it is not install it first.
+```sh
+git --version
+```
+
+Create user to run gitea (ex. `git`)
+```sh
+adduser \
+   --system \
+   --shell /bin/bash \
+   --gecos 'Git Version Control' \
+   --group \
+   --disabled-password \
+   --home /home/git \
+   git
+```
+
+### Create required directory structure
+
+```sh
+mkdir -p /var/lib/gitea/{custom,data,indexers,public,log}
+chown git:git /var/lib/gitea/{data,indexers,log}
+chmod 750 /var/lib/gitea/{data,indexers,log}
+mkdir /etc/gitea
+chown root:git /etc/gitea
+chmod 770 /etc/gitea
+```
+
+**NOTE:** `/etc/gitea` is temporary set with write rights for user `git` so that Web installer could write configuration file. After installation is done it is recommended to set rights to read-only using:
+```
+chmod 750 /etc/gitea
+chmod 644 /etc/gitea/app.ini
```

Comment:
I wouldn't give read access to all, since app.ini holds some sensitive data!

Reply:
`chmod 750 /etc/gitea` does allow only root:git access to contents inside this directory

Reply:
Missed that totally!